lib/dpl/provider/now.rb

```diff
@@ -0,0 +1,136 @@
+module DPL
+    class Provider
+      class Now < Provider
+        npm_g 'now'
+  
+        def directory
+          File.expand_path( (context.env['TRAVIS_BUILD_DIR'] || '.' ) + '/' + (options[:directory] || '') )
```

Comment:
`TRAVIS_BUILD_DIR` should be defined in all our builds. If it is not, then there is something else that's grossly concerning.

`options.fetch(:directory, '')` makes the intent clearer.